Dockerfile

```diff
@@ -92,7 +92,7 @@ RUN if [[ -z "$KUBECTL" ]] ; then echo KUBECTL not requested ; else \
         rpm -ivh --nodeps azure-cli-*.rpm && \
         rm -f azure-cli-*.rpm && \
         pip install -r /runner/deps/python_azure.txt && \
-        pip install azure-cli-core==2.30.0 --upgrade \
+        pip install azure-cli-core==2.37.0 --upgrade \
```

Comment:
@wmudge requires significant testing with Azure; we may with to continue with 2.30